.circleci/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# save current path
+REPO_PATH=$PWD
+
+# Save some useful information
+SHA=`git rev-parse --verify HEAD`
+
+# Clone the existing gh-pages for this repo into $HOME/gh-pages/
+# Create a new empty branch if gh-pages doesn't exist yet (should only happen on first deploy)
+cd $HOME
+git config --global user.email "circleci@circle-ci.org"
+git config --global user.name "circle-ci"
+git clone --quiet --branch=master https://${GH_PAGES_DEPLOY}@github.com/camel-tooling/camel-lsp-client-eclipse-update-site.git gh-pages > /dev/null
```

Comment:
`GH_PAGES_DEPLOY` token has been provided in Circle Ci project settings https://app.circleci.com/settings/project/github/camel-tooling/camel-lsp-client-eclipse/environment-variables?return-to=https%3A%2F%2Fcircleci.com%2Fdashboard